aws/resource_aws_spot_fleet_request.go

```diff
@@ -653,6 +681,33 @@ func resourceAwsSpotFleetRequestStateRefreshFunc(d *schema.ResourceData, meta in
 	}
 }
 
+func resourceAwsSpotFleetRequestFulfillmentRefreshFunc(d *schema.ResourceData, meta interface{}) resource.StateRefreshFunc {
```

Comment:
Sounds good to me. I just copied the whole function from above. Also I already thought about merging these two somehow since they look almost the same.